test/test-rotatesArrayOfCoordinates.js

```diff
@@ -0,0 +1,334 @@
+var assert = require('chai').assert;
+var tetris = require('../tetris')
+describe("Incoming array is empty", function () {
+    if ("Array is empty", function () {
+        let arrayOfCoordinates = [];
+        let pivot = tetris.getCoordinates(`
```

Comment:
you don't need such a big field in this case, even non-graphic representation will work